src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div id="name-request-summary">
+    <v-row id="name-request-info">
+      <v-col>
+        <label>
+          <strong>Name Request</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li class="name-request-title">
+            <strong>{{ getBusinessIdentifier }}</strong> - {{ getNameRequestDetails.approvedName}}
+          </li>
+          <li class="mt-4">Entity Type: {{ entityTypeDescription() }}</li>
+          <li> Request Type: {{ requestType() }}</li>
+          <li class="mt-4">Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>Status: {{ getNameRequestDetails.status }}</li>
+          <li>Condition/Consent: {{ conditionConsent() }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+    <v-row id="name-request-applicant-info">
+      <v-col>
+        <label>
+          <strong>Name Request Applicant</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ getNameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ getNameRequestApplicant.phoneNumber }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { getName } from 'country-list'
+
+// Enums
+import { NameRequestStates } from '@/enums'
+
+// Interfaces
+import { GetterIF, NameRequestDetailsIF, NameRequestApplicantIF } from '@/interfaces'
+
+// Mixins
+import { DateMixin } from '@/mixins'
+
+@Component
+export default class NameRequestInfo extends Mixins(DateMixin) {
+  // Global getters
+  @Getter isEntityType!: GetterIF;
+  @Getter isTypeBcomp!: GetterIF;
+  @Getter isTypeCoop!: GetterIF;
+  @Getter getBusinessIdentifier!: GetterIF;
+  @Getter getNameRequestDetails!: NameRequestDetailsIF;
+  @Getter getNameRequestApplicant!: NameRequestApplicantIF;
+
+  /** The entity title  */
+  private entityTypeDescription () :string {
+    if (this.isTypeBcomp) {
+      return 'BC Benefit Company'
+    } else if (this.isTypeCoop) {
+      return 'BC Cooperative Association'
+    }
+
+    return ''
+  }
+
+  /** The request type */
+  private requestType () : string {
+    return 'New Business'
+  }
+
+  /** Return formatted expiration date */
+  private formattedExpirationDate () : string {
+    return this.toReadableDate(this.getNameRequestDetails.expirationDate)
+  }
+
+  /** Return consent received string by checking if conditional and if consent has been received */
+  private conditionConsent (): string {
+    if (this.getNameRequestDetails.status === NameRequestStates.CONDITIONAL) {
+      return this.getNameRequestDetails.consentFlag === 'R' ? 'Received' : 'Not Received'
+    }
+
+    return ''
```

Comment:
Should this return "Not applicable", or hide this line, if no consent is not required? Ask Scott...

Reply:
Checking with Scott.

Reply:
Great. Feel free to merge this PR after this conversation is resolved.

Reply:
Conclusion was to hide the consent field when it isn't relevant. 
- added logic in the template to only show consent field when relevant
- updated tests